Sources/App/configure.swift

```diff
@@ -28,9 +28,11 @@ private let jiraProjects = [
     "PRO"   : 16980, // Professional Services
     "PRSCR" : 16840, // Prescriptions
     "REFER" : 17244, // Referrals (next-gen)
+    "RM"    : 1704,  // Real-Time Matching
```

Comment:
Is that correct ? Seems like a low ID value for a recent board

Reply:
It is indeed wrong, BRB.